### PR TITLE
Better Admin Logs

### LINKED ISSUE #9593: [BUG] Automod does not log player deaths.
### There is no other issue reporting the same problem.

- [X] I have searched the existing issues

### Game Version

Upstream/Develop (2/6/2023)

### I was

Server

### OS

Windows 8, 10, 11

### What happened?

When players die by non-antags; their deaths aren't logged in admin alerts.

### What should have happened instead?

When `OnDeathActions()` is called, it should process who was the last person to attack the dead player and alert admins about it if they're non-antags or if the player did that to themselves.

### Steps to reproduce the issue

1. Kill yourself while having access to admin alerts.
2. Watch as no alerts happen.
3. Kill another player while not having an antagonist role.
4. No alerts happen still

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by MaxIsJoe**
fixes #9593 

The new implementation stores all important gameplay related logs, and lets admins access those logs in game.
The current iteration is not meant to be pretty, as it favors functionality over UX for admins currently; this will be further expanded upon later in the future once this system is battle tested.

### What's currently missing and will be added later:

- Admin actions (Such as teleporting to cords in logs, gibbing trouble players instantly, etc)
- Notifications for important logs.

### Changelog:

CL: [New] The game will now store gameplay related round logs for admins to access.
CL: [New] A lot of in-game actions are logged now.
CL: [Improvement] Rounds are now properly numbered.
CL: [Fix] Fixes a missing bowink sound for admins.

### CODE REVIEW

File: UnityProject/Assets/Scripts/Core/Admin/Logs/Interfaces/IAdminHumanReadableLogEntryConverter.cs [hunk 1]
```diff
@@ -0,0 +1,7 @@
+﻿namespace Core.Admin.Logs.Interfaces
+{
+	public interface IAdminHumanReadableLogEntryConverter
```
Comment (Bod9001) [inline UnityProject/Assets/Scripts/Core/Admin/Logs/Interfaces/IAdminHumanReadableLogEntryConverter.cs:3]: IAdminHumanReadableLogEntryConverter?

Comment (Bod9001) [reply UnityProject/Assets/Scripts/Core/Admin/Logs/Interfaces/IAdminHumanReadableLogEntryConverter.cs:3]: as empty

Comment (MaxIsJoe) [reply UnityProject/Assets/Scripts/Core/Admin/Logs/Interfaces/IAdminHumanReadableLogEntryConverter.cs:3]: yeah its meant to be like this

File: UnityProject/Assets/Scripts/UI/Systems/AdminTools/AdminChatWindows.cs [hunk 2]
```diff
@@ -75,7 +75,7 @@ public void ResetAll()
 			mentorPlayerChat.ClearLogs();
 			adminToAdminChat.ClearLogs();
 			playerPrayerWindow.ClearLogs();
-			adminLogWindow.ClearLogs();
+			//adminLogWindow.ClearLogs();
```
Comment (Bod9001) [inline UnityProject/Assets/Scripts/UI/Systems/AdminTools/AdminChatWindows.cs]: intentional?

File: UnityProject/Assets/Scripts/UI/Systems/AdminTools/AdminLogs/AdminLogEntryUI.cs [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+		public string LogText => entry?.Log;
+
+		public void Setup(LogEntry newEntry)
+		{
+			entry = newEntry;
+			logInfo.text = entry.Log;
+			logTime.text = newEntry.LogTime.ToLocalTime().ToLongTimeString();
+			gameObject.SetActive(true);
+			//gibButton.onClick.AddListener(GibRequest);
+			//teleportButton.onClick.AddListener(TeleportTo);
```
Comment (Bod9001) [inline UnityProject/Assets/Scripts/UI/Systems/AdminTools/AdminLogs/AdminLogEntryUI.cs:29]: code?

File: UnityProject/Assets/Scripts/UI/Systems/AdminTools/AdminChatWindows.cs (hunk 2 above)
Comment (MaxIsJoe) [reply UnityProject/Assets/Scripts/UI/Systems/AdminTools/AdminChatWindows.cs]: there is no ClearLogs() function currently as entries get nuked whenever a page gets loaded, so this should probably be removed.

File: UnityProject/Assets/Scripts/UI/Systems/AdminTools/AdminLogs/AdminLogEntryUI.cs (hunk 3 above)
Comment (MaxIsJoe) [reply UnityProject/Assets/Scripts/UI/Systems/AdminTools/AdminLogs/AdminLogEntryUI.cs:29]: WIP

**Review by Bod9001 — APPROVED** [review]

**Pull request merged**